packages/cli/src/scanner/page.ts

```diff
@@ -39,25 +43,29 @@ export class Page {
             deviceScaleFactor: 1,
         });
 
-        const gotoUrlPromise = this.puppeteerPage.goto(url, { waitUntil: ['load'], timeout: 120000 });
-        const networkLoadTimeoutInMilleSec = 30000;
-        const waitForNetworkLoadPromise = this.puppeteerPage.waitForNavigation({
-            waitUntil: ['networkidle0'],
-            timeout: networkLoadTimeoutInMilleSec,
-        });
-
-        let response;
+        // separate page load and networkidle0 events to bypass network activity error
+        const gotoUrlPromise = this.puppeteerPage.goto(url, { waitUntil: 'load', timeout: this.pageNavigationTimeoutMsecs });
```

Comment:
so if we wait in here, why do we still need 'waitForPageToCompleteRendering'?

Reply:
some pages will have active javascript after load event that affect page rendering and as result color contrast detection. This results in false positive accessibility issues from axe.